### ISSUE TITLE
AQ War Effort

### ISSUE THREAD

**Issue by Grimfeather** [ZhengPeiRu21/mod-individual-progression#159]
When I set mod-individual-progression to 3, which I believe is tier 4 pre-AQ, the AQ gate is still closed.
The scepter quest line is available in Cenarion Hold, but the war effort npcs in Ironforge and Orgrimmar are missing.
I've never taken part in the war effort, so I'm not sure how this works, but I think something is wrong.
You normally phase stuff out, it would always show up in GM mode 
and it would be available and visible when the player reaches the correct progression level. 
But this time nothing is there.

In Ironforge the collection tables are there, but the npcs (like Sergeant Stonebrow) are not.
In Orgrimmar I can't find anything related to the war effort in the valley of spirits, no npcs, no tables, nothing.

some info that may be needed:
- I did run zz_optional_aq_quest_nerf.sql
- in individualProgression.conf, IndividualProgression.RequirePreAQQuests = 1

**Comment by ZhengPeiRu21** [comment]
The AQ gate being closed is correct behavior for that phase. The gate will open once the questline to open it is complete and the gong has been rung. Please note that the AQ questline is separate from the AQ War Effort event, which is a server-wide effort to turn in resources and is not part of the module at the moment, since it would need to be re-designed as an individual player event. The questline begins with this quest: https://www.wowhead.com/classic/quest=8286/what-tomorrow-brings

Please let me know if you have any other questions or run into any issues.

**Comment by Grimfeather** [comment]
Ok, I misunderstood. I know the gate being closed is correct.
The thing I wanted to report were the missing npc's for the war effort.

I thought the zz_optional_aq_quest_nerf file also covered the AQ war effort hand in quests. 
I now realize it only effects the reputation grind quests.
If the AQ war effort isn't part of this module (yet) then this will be a request to add it.

Even setting it to 10 hand ins for each resource would already be great.
And then make it so finishing it on one side, horde or alliance, is enough to complete the entire thing.

It does require skinning, mining and herbalism, which if you play this alone you will not have.
I suggest to combine those 3 hand ins so the npcs will ask for ore, leather OR herbs.
Much like the thorium brotherhood does it with their reputation quest (heavy leather, kingsblood or iron bars)

for horde you then combine
peacebloom, tin bar and heavy leather
firebloom, mithril bar and rugged leather

for alliance
stranglekelp, iron bar and light leather
arthas tears, thorium bar and medium leather

and for both factions
copper bar, purple lotus and thick leather

That would give the player the entire AQ event experience.

**Comment by kekeshgreen** [comment]
I would just like to endorse this request. The War Effort is a really fun part of the AQ experience :)

This module is incredible overall though. Keep up the good work 💯
